packages/notices/src/store/actions.ts

```diff
@@ -303,9 +281,12 @@ export function createWarningNotice( content, options ) {
  *};
  * ```
  *
- * @return {Object} Action object.
+ * @return Action object.
  */
-export function removeNotice( id, context = DEFAULT_CONTEXT ) {
+export function removeNotice(
+	id: string,
+	context = DEFAULT_CONTEXT
```

Comment:
Not sure if there's a WPCS recommendation somewhere, but personally I'd provide an explicit type definition here (and elsewhere) instead of inferring types from a constant (let alone an imported one).

- It's more self-documenting.
- Makes it easier to guard internally against an unintentional backcompat type break somewhere else in the codebase, or in a similar vein spot discrepancies between the doc-block and the actual method.
- (Also, anecdotally so this could be a skills issue on my part, but type inference can sometimes be flaky in IDEs)